src/context/GameContext.js

```diff
@@ -0,0 +1,31 @@
+import { createContext, useContext, useState } from 'react';
+const GameContext = createContext();
+
+const GameProvider = ({ children }) => {
+  const [squares, setSquares] = useState(Array(9).fill(null));
+  const [currentPlayer, setCurrentPlayer] = useState();
+  const [active, setActive] = useState();
+  const [gameMessage, setGameMessage] = useState();
```

Comment:
or this one — instead you are using something called "value" which you are not treating like a state variable. this looks very much like you read the instructions without understanding the purpose of the things you were asked to do, doing what was technically asked without actually following through. despite this lack of understanding, you did not make any TA requests. perhaps when you hit a point where you don't understand something, you might be better served by using the resources available to you in this program and asking for help